app/scripts/migrations/053.test.js

```diff
@@ -64,7 +64,10 @@ describe('migration #53', function () {
       TransactionController: {
         transactions: [
           { type: TRANSACTION_TYPES.CANCEL, txParams: { foo: 'bar' } },
-          { type: TRANSACTION_TYPES.SENT_ETHER, txParams: { foo: 'bar' } },
+          {
+            type: TRANSACTION_TYPES.SENT_ETHER,
```

Comment:
Was this a lint related change? If SENT_ETHER doesn't exist anymore this might be a lint issue

Reply:
replaced with local const in the test file.